### PR TITLE
[TR] PIM-4759: cache the default image

### PR DESCRIPTION
**Pull request by jjanvier**
| Q                  | A
| ------------------ | ---
| Migration script?  | -
| Behats             | 
| Specs & Static     | Y
| Changelog updated  | -

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Controller/FileController.php [hunk 1]
```diff
@@ -19,6 +20,8 @@
  */
 class FileController extends Controller
 {
+    const DEFAULT_IMAGE_KEY=  '__default__';
```
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/Controller/FileController.php]: phpdoc

Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Controller/FileController.php]: space before = and extra space after

**Comment by nidup** [comment]
GTM

**Comment by grena** [comment]
gtm after comments

**Comment by phaseinducer** [comment]
GTM

**Pull request merged**